scripts/swo_parser.py

```diff
@@ -0,0 +1,182 @@
+#
```

Comment:
this is a python script i found on github. It works as expected

Reply:
maybe link to the original repo

Reply:
also python version requirements?